code-samples/eventing/bookstore-sample-app/solution/solution.sh

```diff
@@ -71,7 +76,7 @@ export KO_DOCKER_REPO=$REGISTRY_HOST/$REGISTRY_PORT
 echo ""
 echo "📦 Installing Camel-K..."
```

Comment:
It seems like $REGISTRY_PORT value cannot be empty here. If it is, the script will exit with error. I haven't used the local registry yet, so couldn't comment on this too much. Originally in the old version of tutorial, the remote registry address is requested.

https://github.com/dprotaso/knative-docs/blob/1ec7a2b8807bd23fcf3c217e0adc5634b5e2d0bf/code-samples/eventing/bookstore-sample-app/solution/solution.sh#L63-L75

Reply:
It looks like these variables are read in line 62/63

Reply:
Yes, as github doesn't allow me to comment on that line 😅

Also, since now we are officially recommend users to use KinD, the port and registry could have a default value.

"🔑 Enter the registry port:" can be updated to "🔑 Enter the registry port (e.g. 5001 for KinD local registry):"